config.toml.in

```diff
@@ -7,6 +7,8 @@
 # The storage section controls the configuration of the backend.
 [storage]
 type = "file"
+auto_backup = true
```

Comment:
Should we move `auto_backup` and `max_backups` to a new `[storage.backup]` / `[general.backup]` section?